lib/transmuxer_common.js

```diff
@@ -0,0 +1,944 @@
+/**
+ * mux.js
```

Comment:
How would you feel about breaking this up by format? "common" isn't bad but `mp2t`, `h264`, and `aac` might be better descriptions of the content.